documentation/book/ref-storage-jbod.adoc

```diff
@@ -3,20 +3,26 @@
 // assembly-storage.adoc
 
 [id='ref-jbod-storage-{context}']
-= JBOD storage
+= JBOD storage overview
 
-The JBOD storage allows you to provide more volumes to the Kafka brokers to store data.
-As well as providing more storage, using multiple disks can improve performance.
+You can configure {ProductName} to use Just a Bunch of Disks (JBOD), a data storage configuration in which multiple disks are combined to create a single logical disk, or volume. JBOD is one approach to providing increased data storage for Kafka brokers. It can also improve performance.
```

Comment:
in the other commit, you used the other way around ... JBOD (Just a Bunch of Disks), isn't it better to be consistent?

Reply:
Yes! Be Consistent! :) But `Just a Bunch of Disks (JBOD)` is the order it should be displayed in. So I'd change the other section referenced.

Reply:
@ncbaratta tbh I think that all developers/engineers refer to it as JBOD more than "Just a Bunch of Disks". I would refer it as JBOD and the having the acronym explained in parenthesis ... so quite the opposite you said :P

Reply:
If most developers just know what it is then I'd remove the spelled out name. If people don't refer to it that way, then we don't have to define it at all.

Reply:
Maybe we can remove the spelled out name here but I would still have it in the `assembly-storage.adoc` file.